aten/src/TH/THAllocator.cpp

```diff
@@ -136,15 +138,21 @@ THMapAllocator::THMapAllocator(WithFd, const char *filename, int fd, int flags,
       AT_ERROR("TH_ALLOCATOR_MAPPED_FROMFD not supported on Windows");
     }
 
+    // Shadowing
+    const wchar_t *filename;
```

Comment:
Maybe refactor a little bit to avoid code duplication?

Reply:
I tried, but there's a parameter named `filename`. The shadowing is only allowed inside a `{}`. So....got any idea?

Reply:
I don't mind, the duplication is not that bad